src/dev/build/lib/fs.js

```diff
@@ -186,10 +186,10 @@ export async function untar(source, destination, extractOptions = {}) {
   ]);
 }
 
-export async function compress(type, options = {}, source, destination) {
+export async function compress(type, options = {}, source, destination, createRootFolder) {
   const output = fs.createWriteStream(destination);
   const archive = archiver(type, options);
-  const name = source.split(sep).slice(-1)[0];
+  const name = (createRootFolder ? source.split(sep).slice(-1)[0] : false);
```

Comment:
Can we please add another property to the `options` object rather than a separate boolean arg?

Reply:
Definitely, pushed https://github.com/elastic/kibana/pull/37258/commits/009601987584357ad2243520a6ec44e0acb4d6ab